support-workers/src/main/scala/com/gu/emailservices/EmailFields.scala

```diff
@@ -4,13 +4,18 @@ import com.gu.i18n.Currency
 import com.gu.salesforce.Salesforce.SfContactId
 import com.gu.support.promotions.Promotion
 import com.gu.support.workers.{BillingPeriod, User}
-import io.circe.generic.auto._
+import io.circe.Encoder
+import io.circe.generic.semiauto._
 import io.circe.syntax._
 
 case class EmailPayloadContactAttributes(SubscriberAttributes: Map[String, String])
 case class EmailPayloadTo(Address: String, ContactAttributes: EmailPayloadContactAttributes)
-case class EmailPayload(To: EmailPayloadTo, DataExtensionName: String, SfContactId: Option[String], IdentityUserId: Option[String]) {
-  lazy val jsonString: String = this.asJson.toString
```

Comment:
this file used to use the "auto" generated encoders, I assume it was because we were serialising based on `this` rather than a compile time known type.  Given that uses shapeless magic, and we don't use them anywhere else, and it wasn't necessary to apply it to `this` anyway, I just converted for consistency.